tox.ini

```diff
@@ -4,10 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py{27,34,35,36,py}-pytest30-supported-xdist,
-          py{27,35}-pytest30-unsupported-xdist,
-          py{27,34,py}-pytest31-supported-xdist
-          py{27,34,py}-pytest34-supported-xdist
+envlist = py{27,34,35,36,37,py}-pytest39-{supported,unsupported}-xdist
```

Comment:
I think we could drop Python versions. Does it make sense to test other Python versions than 2.7 and 3.7 (and maybe 3.4)? Is there really anything that could break in between?

Also, I guess we want to test pytest310 now

Reply:
Dropping versions is fine in general, but the envlist is just for "tox -l", it is not being run completely, so it can/should have all supported versions.

> Also, I guess we want to test pytest310 now

:+1: